api/routes/api_news.py

```diff
@@ -39,11 +40,26 @@ def get_sport_news_endpoint(sport_type):
 @cache.cached(timeout=60*3)
 def get_popular_news_endpoint():
     try:
-        recent_news = get_popular_news(5, session)
-        return get_good_response(recent_news)
+        pouplar_news = get_popular_news(5, session)
+        return pouplar_news
     except Exception as e:
         response = {"error in service": str(e)}
         return get_error_response(response, 500)
+    
+@news_app.route('/article', methods=['POST'])
+def specific_article():
+
+    try:
+        article = request.get_json()
+        news_id=article['blob_id']
```

Comment:
If we only care about the 'blob_id' wouldn't it make more sense to pass it alone? Why waste both traffic and RAM strong a whole bunch of useless bytes (I mean the article text) for however little time